addons/gearinfo/functions/fnc_getModelOptions.sqf

```diff
@@ -1,6 +1,6 @@
 #include "script_component.hpp"
 
-params ["_classRoot", "_model", ["_modelDefinition", configNull], ["_kind", "options"]];
+params ["_classRoot", "_model", ["_modelDefinition", objNull], ["_kind", "options"]];
```

Comment:
`_modelDefinition` is expected to be a config object, we should keep `configNull` for type consistency